### ISSUE TITLE
Description of Yeo atlas has wierd type

### ISSUE THREAD

**Issue by banilo** [nilearn/nilearn#1635]
Small wrinkle, the description for the yeo atlas in nilearn.datasets seems to be byte string typed, which makes it difficult to print on the fly:

<img width="961" alt="screenshot 2018-04-24 09 22 10" src="https://user-images.githubusercontent.com/3907396/39173589-75ac31a8-47a5-11e8-8db4-b64ef4584a10.png">

**Comment by emdupre** [comment]
I believe the byte string is from python2 , where str and bytes types are equivalent. On my system, looking at `yeo.description` in py27 has no leading `b` and the `print` function does pretty printing, as expected. If you want to render it properly in py3, decoding works: 
```
yeo = nilearn.datasets.fetch_atlas_yeo_2011()
print(yeo.description.decode("utf-8"))
```
Or, if there's interest in updating atlas descriptions in the code base, happy to open a PR !

**Comment by jeromedockes** [comment]
most atlas fetchers return bytes:
```
>>> from nilearn.datasets import atlas
... import warnings
... for f in atlas.__dict__:
...     if 'fetch_atlas' in f and 'harvard' not in f:
...         args = ('ba',) if 'talairach' in f else ()
...         with warnings.catch_warnings():
...             warnings.simplefilter('ignore')
...             print('{:>35} : {}'.format(
...                 f, type(getattr(atlas, f)(*args).description)))
...                 
             fetch_atlas_allen_2011 : <class 'bytes'>
         fetch_atlas_destrieux_2009 : <class 'str'>
              fetch_atlas_talairach : <class 'str'>
               fetch_atlas_yeo_2011 : <class 'bytes'>
         fetch_atlas_surf_destrieux : <class 'bytes'>
             fetch_atlas_smith_2009 : <class 'bytes'>
   fetch_atlas_basc_multiscale_2015 : <class 'bytes'>
                    fetch_atlas_aal : <class 'bytes'>
                   fetch_atlas_msdl : <class 'bytes'>
          fetch_atlas_craddock_2012 : <class 'bytes'>
>>> 

```

most dataset fetchers too, but not all of them.

it would indeed be great if all fetchers had the same behaviour (and description should probably be a unicode string, i.e. it is the fetcher's responsibility to decode it).

**Comment by jeromedockes** [comment]
related to #1489

**Comment by NicolasGensollen** [comment]
Solved by #2655
